templates/checkout/_partials/steps/addresses.tpl

```diff
@@ -42,12 +42,12 @@
         }
       </div>
 
-      <a href="{$new_address_delivery_url}">{l s='Add another address'}</a>
+      <a href="{$new_address_delivery_url}">{l s='Add another address' d='Shop.Theme.Actions'}</a>
```

Comment:
Sorry it's not about domains, but to avoid duplicates, could we change the string to "Add a new address"? Thanks!